package.json

```diff
@@ -9,7 +9,7 @@
     "start": "ts-node server/server.ts",
     "build": "webpack --mode production",
     "test": "jest",
-    "cypress run": "cypress open mainTest.e2e-test",
+    "cypress-run": "cypress run mainTest.e2e-test",
```

Comment:
you can name it test:e2e.

It will be easier to understand for people who don't know cypress what is going up here.
Also it is like polymorphism for configuration - if we ever change the testing framework the name stays the same - only implementation changes.